pennylane/drawer/tape_mpl.py

```diff
@@ -259,7 +259,8 @@ def _tape_mpl(tape, wire_order=None, show_all_wires=False, decimals=None, *, fig
     if fontsize is not None:
         drawer.fontsize = fontsize
 
-    drawer.label(list(wire_map), text_options=label_options)
+    if label_options != "off":
```

Comment:
Minor: is it possible to delay this `drawer.label` to the end, so the two `if label_options` checks can be merged into one?

Reply:
Moving it causes errors, but pulling up the cropping works I think.